src/batch/src/executor/test_utils.rs

```diff
@@ -14,19 +14,57 @@
 
 use std::collections::VecDeque;
 use std::future::Future;
+use std::sync::Arc;
```

Comment:
Move this file to `benches` folder, I think it's used by benches only.

Reply:
It seems some unit tests of batch executors need utilities in this file.

Reply:
> Move this file to `benches` folder, I think it's used by benches only.

It seems some unit tests of batch executors need utilities in this file.